perf-tests/clusterloader2/pkg/framework/framework.go

```diff
@@ -158,14 +188,34 @@ func (f *Framework) deleteNamespace(namespace string) error {
 func (f *Framework) DeleteAutomanagedNamespaces() *errors.ErrorList {
 	var wg wait.Group
 	errList := errors.NewErrorList()
-	for i := 1; i <= f.automanagedNamespaceCount; i++ {
-		name := fmt.Sprintf("%v-%d", f.automanagedNamespacePrefix, i)
-		wg.Start(func() {
-			if err := f.deleteNamespace(name); err != nil {
-				errList.Append(err)
-				return
-			}
-		})
+	automanagedNamespacesList, staleNamespaces, err := f.ListAutomanagedNamespaces()
+	if err != nil {
+		errList.Append(err)
+		return errList
+	}
+	if len(automanagedNamespacesList) > 0 {
```

Comment:
Nit: it seems to me the code from line 196 to 218 can be simplified as :

namespacesToDelete := append(automanagedNamespacesList, staleNamespaces)
for _, nsName := namespacesToDelete {
       wg.Start(func() {
				if err := f.deleteNamespace(nsName); err != nil {
					errList.Append(err)
					return
				}
			})
}